src/components/ButtonPlusMin.jsx

```diff
@@ -0,0 +1,11 @@
+
+export default function ButtonPlusMin ({buttonText, buttonActie, buttonDisabled}) {
```

Comment:
Ik vind dit nu een doorgeefluik. Het bespaard geen code, omdat alles wat hier binnenkomt in de functie direct doorgegeven word aan het button html element. Je had dan net zo goed direct het button html element kunnen gebruiken.